main/core/cache.ts

```diff
@@ -15,6 +15,10 @@ export class InstanceCache implements ICache {
         this.instanceMap.clear();
     }
 
+    public all(): Array<any> {
```

Comment:
what does this do? jsdoc comments would be good.
`cache.all()` to me implies that it would cache everything (whatever that means). Instead it seems to be retunring an adday of stuff rather than caching anything.